packages/manager/modules/pci/src/projects/project/storages/databases/database/users/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,28 @@
+{
+  "pci_databases_users_title": "Users & Roles",
+  "pci_databases_users_username_label": "Nom d'utilisateur",
+  "pci_databases_users_description_label": "Description",
+  "pci_databases_users_role_label": "Rôles",
+  "pci_databases_users_createdAt_label": "Date de création",
+  "pci_databases_users_status_label": "Status",
```

Comment:
```suggestion
  "pci_databases_users_status_label": "Statut",
```

Reply:
Corrected here: https://github.com/ovh/manager/pull/6090/commits/548d9e809657d6083ad9d5b22cd0a05986765743
Thank you !